serverdock/Dockerfile

```diff
@@ -1,19 +0,0 @@
-FROM phusion/baseimage
```

Comment:
until we have a replacement for `gsn-docker-relay`, you cannot remove it.
(and even if we do remove it - its going to be a separate commit from "StakeManager"